src/python/pants/goal/run_tracker.py

```diff
@@ -191,6 +197,53 @@ def retrieve_logs(self) -> List[str]:
 
         return output
 
+    def _report(self) -> None:
+        total_run_time = "Unknown"
+        try:
+            total_run_time = str(self._total_run_time)
```

Comment:
Can `str()` ever fail?

Reply:
For classes with custom `__str__` or `__repr__` methods which raise exceptions.

Reply:
Unlikely to be the case for `self._total_run_time`, though. :)